flexmeasures/ui/static/openapi-specs.json

```diff
@@ -7,7 +7,7 @@
     },
     "termsOfService": null,
     "title": "FlexMeasures",
-    "version": "0.31.0"
+    "version": "0.28.1"
```

Comment:
This should not happen usually. 

Can you check if "v0.30.0" and "v0.31.0.dev0" are in the output of `git tag` on your PC?